examples/hello-compute/main.rs

```diff
@@ -29,14 +29,20 @@ fn main() {
     let cs = include_bytes!("shader.comp.spv");
     let cs_module = device.create_shader_module(&wgpu::read_spirv(std::io::Cursor::new(&cs[..])).unwrap());
 
-    let staging_buffer = device
-        .create_buffer_mapped(
-            numbers.len(),
-            wgpu::BufferUsage::MAP_READ
+    let staging_buffer = device.create_buffer(
```

Comment:
why are we no longer using `create_buffer_mapped`?

Reply:
Mostly just so I could make sure both read and write worked. None of the samples have any real use-case for non-create map_write.